packages/schema/src/interfaces.ts

```diff
@@ -45,7 +45,13 @@ export interface IJGISSourceDocChange {
   }>;
 }
 
+export interface ISelection {
+  type: 'layer' | 'source';
```

Comment:
Can you think of use cases where we can select both layers and sources?

Reply:
You mean selecting both layers and sources at the same time? 

Actually I'm not using the type entry anymore in my implementation, I can remove it.

Reply:
> You mean selecting both layers and sources at the same time?

Yes, this is what I meant.
We will probably want to be able to select several sources or several layers (to delete them, or maybe update properties).
But I don't know if we want to select layers and sources at the same time, this is why I wonder if this type is need.

Reply:
This entry was just some metadata information on the selection (we could add more metadata if we want in the future), but we can already infer from the `id` of the selected object if it's a layer or a source. So I should probably remove it.

So technically my code allows selecting both layers and sources at the same time. We'll see if we actually need this if we have use cases.